### PR TITLE
CU-8695pfuj4_Calendar-Tab

### PR DESCRIPTION
**Pull request by zippiL**

### CODE REVIEW

**Comment by benshabatnoam** [comment]
Task linked: [CU-8695pfuj4 Calendar Tab](https://app.clickup.com/t/8695pfuj4)
Task linked: [CU-8695uuj68 Get minyans by the date](https://app.clickup.com/t/8695uuj68)

File: server/router/minyanListRouter.ts [hunk 1]
```diff
@@ -3,25 +3,30 @@ import MinyanListController from "../controller/minyanListController";
...
+  MinyanListController.get(req, res);
+});
 
-export default MinyanListRouter;
\ No newline at end of file
+MinyanListRouter.get(
+  "/getMinyanimByDateType",
+  (req: Request, res: Response) => {
+    console.log("getMinyanimByDateType");
+
```
Comment (SaraCohenNBS) [inline server/router/minyanListRouter.ts:13]: Remove console.log

**Review by SaraCohenNBS — CHANGES_REQUESTED** [review]

**Review by zippiL — COMMENTED** [review]
> Just now I get the point what you mean when you say 'getting directly from the API', that not look so good, no other way? 
i didn't find
If you find it, I would be really happy 😌

File: server/controller/minyanListController.ts [hunk 2]
```diff
@@ -2,9 +2,31 @@ import { Request, Response } from "express";
...
+import mongoose from "mongoose";
+
+// Function to determine if today is Rosh Chodesh
+const isRoshChodesh = async (): Promise<boolean> => {
+  const now = new Date();
+  const hebcalRes = await axios.get(
+    `https://www.hebcal.com/converter?cfg=json&gy=${now.getFullYear()}&gm=${
+      now.getMonth() + 1
+    }&gd=${now.getDate()}&g2h=1`
+  );
```
[the remark this answers is not in the document]
Comment (zippiL) [reply server/controller/minyanListController.ts:15]: i didn't find
If you find it, I would be really happy 😌

File: server/controller/minyanListController.ts [hunk 3]
```diff
@@ -2,9 +2,31 @@ import { Request, Response } from "express";
 import dayjs from "dayjs";
 import MinyanListModel from "../models/minyanListModel";
 import { io } from "../socketio";
+import axios from "axios";
+import mongoose from "mongoose";
+
+// Function to determine if today is Rosh Chodesh
+const isRoshChodesh = async (): Promise<boolean> => {
```
[the remark this answers is not in the document]
Comment (zippiL) [reply server/controller/minyanListController.ts:9]: Where to put it exactly?
We will pass it on.
May you feel comfortable...

File: server/controller/minyanListController.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (SaraCohenNBS) [reply server/controller/minyanListController.ts:16]: 👍

**Pull request merged**